itunes_app_scraper/scraper.py

```diff
@@ -205,21 +209,23 @@ def get_app_details(self, app_id, country="nl", lang="", flatten=True):
 
 		return app
 
-	def get_multiple_app_details(self, app_ids, country="nl", lang=""):
+	def get_multiple_app_details(self, app_ids, country="nl", lang="", sleep=None):
 		"""
 		Get app details for a list of app IDs
 
 		:param list app_id:  App IDs to retrieve details for
 		:param str country:  Two-letter country code for the store to search in.
 		                     Defaults to 'nl'.
 		:param str lang: Dummy argument for compatibility. Unused.
+		:param int sleep: Seconds to sleep after each request to prevent being
+						  temporary blocked. Defaults to None.
 
 		:return generator:  A list (via a generator) of app details
 		"""
 		for app_id in app_ids:
 			try:
 				time.sleep(1)
-				yield self.get_app_details(app_id, country=country, lang=lang)
+				yield self.get_app_details(app_id, country=country, lang=lang, sleep=sleep)
```

Comment:
and this makes it obvious:
`get_multiple_app_details` runs multiple calls to `get_app_details` with a sleep of 1 in between
in addition it is possible to pass further sleeps:
```
loop over app_ids
   sleep
   get_app_details 
```

which translates to:
```
loop over app_ids
   sleep
   call url
   sleep
```
This does no make sense to me

Reply:
Nor to me either. I overlooked the static sleep command. I will fix that.